src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -540,6 +540,7 @@ def load_arguments(self, _):
         c.argument('display_name', arg_type=ts_display_name_type)
         c.argument('description', arg_type=ts_description_type)
         c.argument('version_description', arg_type=ts_version_description_type)
+        c.argument('no_prompt', options_list=['--yes', '-y'], action='store_true', help='Do not prompt for confirmation')
```

Comment:
From your current design, the default behavior will be prompt, which is a breaking change for existing users, especially automation scripts.
If you are sure to make such change, please let me know and mark BREAKING CHANGE

Reply:
That is the intention, title has been updated.